packages/flame/lib/src/assets/images.dart

```diff
@@ -10,12 +10,18 @@ import 'package:flutter/services.dart';
 import '../flame.dart';
 
 class Images {
-  Images({this.prefix = 'assets/images/'})
-      : assert(prefix.isEmpty || prefix.endsWith('/'));
+  Images({String prefix = 'assets/images/'})
+      : assert(prefix.endsWith('/')),
+        _prefix = prefix;
 
-  final String prefix;
+  String _prefix;
   final Map<String, _ImageAssetLoader> _loadedFiles = {};
 
+  set prefix(String value) {
+    assert(value.endsWith('/'));
```

Comment:
An assert message here maybe?

Reply:
Or we just add `/` if it doesn't exist?

Reply:
could be, I just kept the previous behaviour, but I don't think automatically adding the `/` could be considered a breaking change right? Because if anyone would be passing without, they would be receiving an error already

Reply:
should you clear the cache here? otherwise you would get weird inconsistencies for the existing keys when they invalidate

Reply:
> should you clear the cache here? otherwise you would get weird inconsistencies for the existing keys when they invalidate

Good question, the pro of not clearing the cache is that if the user changes the prefix back again, the cache is already there, not sure what would be best in this case